server/thirdparty/integrationConfig.js

```diff
@@ -44,6 +44,24 @@ const IntegrationConfig = {
       },
     }],
   },
+  Github: {
+    icon_path: 'slackIcon.png',
+    Events: [{
+      EventName: 'Create New Issue',
+      EventType: 'Action',
+      OutputsWeGet: {
+        text: '%STRING%',
+        channel: {
+          url: 'https://slack.com/api/conversations.list',
```

Comment:
Why do we need slack channel for github action

Reply:
It is not used. We can get the message from any channel.
Command based action messages are created for separate action.